INSTALL.md

```diff
@@ -60,24 +60,46 @@ You will need:
  * PhysFS
  * oggenc (to generate music files)
 
-On Ubuntu (and probably any other Debian-based system), you can use the following command to install all required packages:
+On Ubuntu (and probably any other Debian-based system) you can use the following commands.
+
+The apt-get update is optional but recommended to use before install packages:
+
+```
+  $ sudo apt-get update
+```
+
+Use the following command to install all required packages:
 ```
-  $ apt-get install build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
+  $ sudo apt-get install --no-install-recommends build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
```

Comment:
`sudo` addon here is alright, some newbie Linux user might not think that apt-get won't run on regular permissions.
`--no-install-recommends` isn't really changing much, I've checked what it changes on fresh 20.04, there was maybe 2MB data less to download, so the effect is negligible unless you deal with really bad internet connection, or you really want to keep your package manager clean and tidy. But it doesn't appear to mess up anything, so I think I'm okay with that.